src/univariate/continuous/chisq.jl

```diff
@@ -42,8 +42,8 @@ params(d::Chisq) = (d.ν,)
 
 ### Conversions
 convert(::Type{Chisq{T}}, ν::Real) where {T<:Real} = Chisq(T(ν))
-convert(::Type{Chisq{T}}, d::Chisq{S}) where {T <: Real, S <: Real} = Chisq(T(d.ν))
-
+Base.convert(::Type{Chisq{T}}, d::Chisq) where {T<:Real} = Chisq{T}(T(d.ν))
```

Comment:
A bit confusing to have `Base.convert` and `convert` side by side, but of course the same given the import

Reply:
I didn't want to touch the `convert(::Type, ::Real...)` methods in this PR since
1. they are broken in so far as they don't ensure that the returned distribution of the desired type, and
2. I think they should be deprecated and eventually be removed.

In particular methods such as `convert(::Type{Normal{T}}, mu, sigma)` seem strange and I don't see a good reason for why one should prefer those over `Normal(mu, sigma)` or `Normal{T}(mu, sigma)`.

Reply:
Then there is no reason to worry about their cosmetics.